doc/RESTAPI_Guidelines.md

```diff
@@ -0,0 +1,118 @@
+# Rules for JSON (in NFS) :
+
+**Rule 1 -** keys should be represented by all small letters.
+<details>
+<summary>Example</summary>
+<pre>
+{
+  name : correct
+  message : correct
+  ID : wrong
+  Address Line : wrong
+}
+</pre>
+</details>
+
+**Rule 2 -** keys should not contain special characters except "_".
+<details>
+<summary>Example</summary>
+<pre>
+{
+  name : correct
+  address_line : correct
+  employee-id : wrong
+  address#1 : wrong
+}
+</pre>
+</details>
+
+**Rule 3 -** multiword keys should separate words using "_" instead of spaces.
+<details>
+<summary>Example</summary>
+<pre>
+{
+  address_line : correct
+  employee_id : correct
+  address line : wrong
+  employee id : wrong
+}
+</pre>
+</details>
+
+**Rule 4 -** NULL in c code should not be mapped to null in json. Absence of a key in json is equivalent to NULL in C code.
```

Comment:
> Absence of a key in json is equivalent to NULL in C code.

Can we make it more clear by specifying something like ( key is undefined if it's NULL in json)

Reply:
Made changes in the example.